build-system/tasks/visual-diff/index.js

```diff
@@ -48,9 +48,19 @@ const {waitUntilUsed} = require('tcp-port-used');
 let puppeteer;
 let percySnapshot;
 
+// CSS injected in every page tested.
+// https://docs.percy.io/docs/percy-specific-css
+const percyCss = [
+  // Loader animation may otherwise be captured in slightly different points,
+  // causing the test to flake.
+  '.i-amphtml-new-loader { display: none; }',
+].join('\n');
+
```

Comment:
There's several ways to include this option globally, like wrapping `percySnapshot()` instead. I have no strong preference about any, so I'll let @danielrozenberg make a choice here.

Reply:
I'm good with this